src/vcpkg/vcpkgpaths.cpp

```diff
@@ -1077,28 +1082,36 @@ namespace vcpkg
                 auto split_line = Strings::split(line, '\t');
                 if (split_line.size() != 2)
                 {
-                    return Strings::format("Error: Unexpected output from command `%s`. Couldn't split by `\\t`.\n%s",
-                                           git_cmd.command_line(),
-                                           line);
+                    Debug::println("couldn't split by tabs");
+                    return msg::format_error(msgGitUnexpectedCommandOutputCmd,
+                                             msg::command_line = git_cmd.command_line())
+                        .append_raw('\n')
+                        .append_raw(line);
                 }
 
                 auto file_info_section = Strings::split(split_line[0], ' ');
                 if (file_info_section.size() != 3)
                 {
-                    return Strings::format("Error: Unexpected output from command `%s`. Couldn't split by ` `.\n%s",
-                                           git_cmd.command_line(),
-                                           line);
+                    Debug::println("couldn't split by spaces");
+                    return msg::format_error(msgGitUnexpectedCommandOutputCmd,
+                                             msg::command_line = git_cmd.command_line())
+                        .append_raw('\n')
+                        .append_raw(line);
                 }
 
                 ret.emplace(split_line[1], file_info_section.back());
             }
             return ret;
         }
 
-        return Strings::format("Error: Couldn't get local treeish objects for ports.\n%s", maybe_output.error());
+        return std::move(maybe_output)
```

Comment:
This error doesn't include the command line (which is no worse than before).

Reply:
I put a todo against myself to change the `Command` and `flatten` family to do a better job of uniformly reporting tool name, full command line, exit code, and output. I'll spot fix the ones you pointed out here. (I do note though that substantially changing the output is the exact opposite of what you asked for before :) )